src/argilla/client/feedback.py

```diff
@@ -50,28 +50,27 @@
     )
 
 FETCHING_BATCH_SIZE = 250
+PUSHING_BATCH_SIZE = 32
+
+_LOGGER = logging.getLogger(__name__)
 
 
 class ValueSchema(BaseModel):
     value: Union[StrictStr, StrictInt]
 
 
 class ResponseSchema(BaseModel):
-    values: Dict[str, ValueSchema]
-    status: Literal["submitted", "missing", "discarded"]
-
-
-class OfflineFeedbackResponse(ResponseSchema):
     user_id: Optional[UUID] = None
+    values: Dict[str, ValueSchema]
+    status: Literal["submitted", "missing", "discarded"] = "submitted"
```

Comment:
The `missing` value is not really a response status. This value is used as a filter to return records without user responses. As the response is not created yet, we cannot link this `missing` status to the response itself. So, you won't have responses with this status; it's a filter value.

Reply:
Thanks @frascuchon then I'll remove this for the moment! Also @nataliaElv asked about this, so now we have the confirmation, thanks 👍🏻